README.md

```diff
@@ -1,18 +1,18 @@
-# DSP APP &mdash; Generic user interface of DaSCH Service Platform
+# DSP-ADMIN &mdash; Admin user interface of DaSCH Service Platform
```

Comment:
It could be confusing when the repo's name is `dsp-app` but the title is `dsp-admin`...